pkg/resiliency/resiliency.go

```diff
@@ -536,7 +536,7 @@ func (r *Resiliency) EndpointPolicy(ctx context.Context, app string, endpoint st
 	var t time.Duration
 	var rc *retry.Config
 	var cb *breaker.CircuitBreaker
-	operationName := fmt.Sprintf("endpoint[%s, %s]", app, endpoint)
+	operationName := "endpoint[" + app + ", " + endpoint + "]"
```

Comment:
While faster, this is arguably less readable; we don't hit this path a lot, do we?

Reply:
We do, it's a very hot path actually